### PR TITLE
[#] Add fine and coarse setting methods to RadianceParameters.

### PR DESCRIPTION
**Pull request by evanweaver**

### CODE REVIEW

File: openstudiocore/src/model/test/RadianceParameters_GTest.cpp [hunk 1]
```diff
@@ -185,4 +185,20 @@ TEST_F(ModelFixture,RadianceParameters_KlemsSamplingDensity_Quantity) {
   EXPECT_EQ(2, value);
 }
 
+TEST_F(ModelFixture, RadianceParameters_FineAndCoarseSettings) {
+  Model model;
+
+  auto radianceParameters = model.getUniqueModelObject<RadianceParameters>();
+
+  // Test CTOR setting coarse
+  EXPECT_TRUE(radianceParameters.isCoarseSettings());
```
Comment (macumber) [inline openstudiocore/src/model/test/RadianceParameters_GTest.cpp]: @evanweaver nice job!

**Comment by macumber** [comment]
@evanweaver looks good except for one thing.  If you set custom parameters, leave the tab and come back, the custom button is correctly checked but all the fields are disabled.  They should be enabled if you open the tab with custom settings

**Comment by macumber** [comment]
These issue are still present https://github.com/NREL/OpenStudio/issues/1549 https://github.com/NREL/OpenStudio/issues/1491 but do not need to get fixed in this pull request

**Comment by evanweaver** [comment]
Thanks @macumber for your review; it should now be done done  :)

**Pull request merged**